jit/v2/JIT_v2.0.0.yaml

```
@@ -1471,7 +1449,7 @@ paths:
                         errorCodeMessage: 'isFYI should be a boolean. `12` provided as value'
         '404':
           description: |
-            In case the implementer does not know of the `portCallServiceID` used in the request (this could be because of a `PUT` request that has not finished processing or simply because the resource does not exist) - it is possible for the implementer to reject the request by returning a `404` (Not Found).
+            In case the implementer does not know of the `portCallServiceID` path parameter (this could be because the resource does not exist) - it is possible for the implementer to reject the request by returning a `404` (Not Found).
```

Not sure if it is helping or required to specify 'path parameter'. I think you can also just leave it out. Like:
```suggestion
            In case the implementer does not know the `portCallServiceID` (e.g. the resource does not exist) - it is possible for the implementer to reject the request by returning a `404` (Not Found).
```

Same for other occurrences below in this PR.

---

done